### ISSUE TITLE
Manager 1.0.0-beta15 with PHP 5.6 does not work anymore 

### ISSUE THREAD

**Issue by markusmilkereit** [contao/contao-manager#227]
The install selection only shows Contao 4.4, which is correct for PHP 5.6 - but when you start the installation process the manager still tries to install 4.5 - which breaks the process completely.

Console ouput repeats that error 3 times, then stops:
> Could not find package contao/managed-edition with version 4.5.*

No problem on another server with PHP 7.1 active, both Contao versions are available, after selecting 4.4 that was the one we got.

**Comment by frontendschlampe** [comment]
I think, it's no problem of the manager. The constraint of Contao Open Source CMS is wrong. Contao 4.5 needs PHP 7, so the error message is correct.

What's the version contraint of Contao Open Source CMS? Is it ```^4.4```? That means: ```>=4.4 <5.0.0```. The right one should be ```4.4.*``` so it will install everything within 4.4 but no 4.5 or higher.

**Comment by markusmilkereit** [comment]
I have no idea where to look up the constraint before Contao is installed - so far I only have the manager on the server. The composer.json file has not been written yet. 

I guess the problem is with the manager selecting the wrong version to install, and either a change on beta 14 or 15, because it worked flawlessly with beta13. I would not know where else to look, I did not dig deeper into the manager yet :)

**Comment by frontendschlampe** [comment]
ah ... now I understand. You make a new installation? That's a bug - you're right. @aschempp have to check for the right constraint in manager, I think.

**Comment by markusmilkereit** [comment]
Yep, fresh install - we're currently checking our older servers for Contao 4.4 compatability, last week installing via manager worked, today it was broken. Thanks for following up :)
